charts/primary-site/templates/deployments/inbox-listener.yaml

```diff
@@ -18,6 +18,9 @@ spec:
     metadata:
       labels:
         app: inbox-listener
+        {{- with .Values.inboxListener.deployment.podLabels }}
+        {{- toYaml . | nindent 8 }}
```

Comment:
This looks possibly fragile to me - with `annotations` for the ingress I opted to handle this with a loop.

Reply:
Hmm, fragile in what way?

I hadn't noticed this was done with annotations, but I think the way it's doing it is a little odd. Unlike env vars that are specified as an array of key/value in kubernetes manifests, annotations and labels are a map. So the approach here allows you to specify it just like you would in your manifests. This is consistent with the approach taken e.g here: https://github.com/bitnami/charts/blob/main/bitnami/external-dns/templates/service.yaml#L7,L10